syncd/syncd_flex_counter.cpp

```diff
@@ -8,6 +8,7 @@ static std::map<std::string, std::shared_ptr<FlexCounter>> g_flex_counters_map;
 static std::set<sai_port_stat_t> supportedPortCounters;
 static std::set<sai_queue_stat_t> supportedQueueCounters;
 static std::set<sai_ingress_priority_group_stat_t> supportedPriorityGroupCounters;
+static std::set<sai_switch_attr_t> supportedSwitchSensors;
```

Comment:
why temperature sensors are in counters ? i think we dont need this implementation here at all, seems like sensors are just another attribute and we can definitely query them

Reply:
as script i dont know, i had in mind orchagent which would query those sensors using sai apis

Reply:
since the load is low, ~8 values in maybe one minute. Moving to orchagent is better. agree with @kcudnik